pkg/printers/internalversion/printers.go

```diff
@@ -953,7 +954,7 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 
 	if pod.DeletionTimestamp != nil && pod.Status.Reason == node.NodeUnreachablePodReason {
 		reason = "Unknown"
-	} else if pod.DeletionTimestamp != nil {
+	} else if pod.DeletionTimestamp != nil && podPhase == api.PodRunning {
```

Comment:
I'm wondering what the status is in the case of a `Pending`, can we add a test? I think it makes sense to still return `Terminating` . Note that a pending deleted pod may take minutes to be in that state if deleted while pulling a large image.

Reply:
Done.